command/runner.go

```diff
@@ -203,3 +223,67 @@ func (r *cmdWaitReadCloser) Close() error {
 	r.closed = true
 	return nil
 }
+
+// Close the pipe and wait until the consumer program exits
+func (p pagerPipe) Close() error {
+	err := p.writer.Close()
+	<-*p.cc
+	return err
+}
+
+func (p pagerPipe) Write(b []byte) (int, error) {
+	return p.writer.Write(b)
+}
+
+func runPager() (*pagerPipe, error) {
+	pager := ""
+	if p := os.Getenv("KUBECOLOR_PAGER"); p != "" {
+		pager = p
+	} else if p := os.Getenv("PAGER"); p != "" {
+		pager = p
+	} else {
+		for _, p := range DefaultPagers {
+			if _, err := exec.LookPath(p); err == nil {
+				pager = p
+				if pager == "less" {
+					pager = "less -R"
+				}
+				break
+			}
```

Comment:
Suggest just having `less` and `more` hardcoded here, without the `DefaultPagers`. Especially since you have to add extra logic on the `less` case anyway.

If you also extract this to it's own function, then it could be reduced quite a lot:

```go
if _, err := exec.LookPath("less"); err == nil {
	return []string{"less", "-R"}
}
```